projects/fsastorefrontlib/src/cms-components/my-account/claim/claim-details/claim-details.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { RoutingService } from '@spartacus/core';
+import { ClaimService } from '../../../../core/my-account/facade';
```

Comment:
Put whole path to the import of service:  '../../../../core/my-account/facade/claim.service';

This kind of import produces problems when lib is built in production mode.